package.json

```diff
@@ -12,19 +12,43 @@
     "api:test": "jest",
     "api:build": "tsc --build --clean && tsc",
     "api:start": "node build/api/server.js",
-    "api:dev": "tsx ./src/api/server.ts",
+    "api:dev": "nodemon --watch src/api --exec tsx ./src/api/server.ts",
     "fe:init": "cd src/frontend && npm install",
     "fe:build": "cd src/frontend && npm run build",
     "fe:dev": "cd src/frontend && npm run dev"
   },
   "keywords": [],
   "contributors": [
-    { "name": "Fergus Bentley", "email": "mail@fergcb.uk", "url": "https://fergcb.uk" },
-    { "name": "Aamirah Patel", "email": "aamirahpatel@hotmail.co.uk", "url": "https://github.com/AamirahP" },
-    { "name": "Joshua Mushaikwa", "email": "jjnotm@gmail.com", "url": "https://github.com/incogzino" },
-    { "name": "Nial", "email": "nial@n1al.com", "url": "https://n1al.com" },
-    { "name": "Srijan Vaddadi", "email": "srijan.vaddadi@gmail.com", "url": "https://srijan.uk" },
-    { "name": "sanju", "email": "ghandhisanju@gmail.com", "url": "https://n1al.com" }
+    {
+      "name": "Fergus Bentley",
+      "email": "mail@fergcb.uk",
+      "url": "https://fergcb.uk"
+    },
+    {
+      "name": "Aamirah Patel",
+      "email": "aamirahpatel@hotmail.co.uk",
+      "url": "https://github.com/AamirahP"
+    },
+    {
+      "name": "Joshua Mushaikwa",
+      "email": "jjnotm@gmail.com",
+      "url": "https://github.com/incogzino"
+    },
+    {
+      "name": "Nial",
+      "email": "nial@n1al.com",
+      "url": "https://n1al.com"
+    },
+    {
+      "name": "Srijan Vaddadi",
+      "email": "srijan.vaddadi@gmail.com",
+      "url": "https://srijan.uk"
+    },
+    {
+      "name": "sanju",
+      "email": "ghandhisanju@gmail.com",
+      "url": "https://n1al.com"
+    }
```

Comment:
Why have the contributors been formatted like this?

Reply:
🤷‍♂️